source/documentation/tools/create-a-derived-table/project-structure.md

```diff
@@ -1,66 +1,723 @@
-      │   │   │       └── prison_safety_and_security__question_answer_fct.yaml  # table property file
-      │   │   └── staging
-      │   │       ├── prison_safety_and_security__nomis_mod_stg.md  # markdown to be rendered in documentation
-      │   │       ├── prison_safety_and_security__nomis_mod_stg.sql  # staging table
-      │   │       └── prison_safety_and_security__nomis_mod_stg.yaml  # table property file
+      │ 
+      ├── staging  # staging domain
+      │   ├── stg_nomis # database
+      │   │      ├── stg_nomis__stg_docs.md # model
+      │   │      ├── stg_nomis__stg_models.yml
+      │   │      ├── stg_nomis__stg_offender.sql
+      │   │      └── stg_nomis__stg_prison.sql
+      │   └── stg_xhibit
+      │          ├── _stg_xhibit.yml
+      │          ├── stg_xhibit__stg_<model_names>.sql
+      │          └── stg_xhibit__stg_court.sql
+      ├── courts
+      │   ├── courts_int
+      │   │
+      │   ├── courts_datamarts
+      │   │
+      │   └── criminal_courts_derived
+      │
+      ├── prison  # domain
+      │   ├── prison_int # database
+      │   │      ├──prison_int__int_<model_names>.sql # model
+      │   │      ...
+      │   ├── prison_datamarts
+      │   │
+
+```
+
+### Models
+
+Finally, you will begin designing / making your models. We suggest a medallion architecture that categorises the model you are making into three distinct layers, `staging`, `intermediate`, `datamarts` and `derived`. Where `datamarts` and `derived` are two halves of the same layer. Each layer represents data that is increasingly well understood, cleaned and standardised. 
+
+The general rule as to what goes into each of these layers comes down to a few considerations. Staging generally is source specific and doesn't change the grain of the data. Then when you want to combine different sources, or change the grain of the data (or other more complicated transformations) then the models belongs in an intermediate database. Finally, the user facing models live in either `_datamarts` or `_derived` databases. The following sections will go into more depth about these categorisations as well as what the medallion architecture is. 
+
+These concepts can be confusing, so if you have any further questions or just want to discuss you project use the [#ask-data-modelling](https://moj.enterprise.slack.com/archives/C03J21VFHQ9) slack channel to reach out. We are happy to answer questions or receive feedback about this guide there!
+
+Finally, it is worth flagging that create-a-derived-table has been evolving since its conception and for that reason project structure guidance, style guidance and best practice have all changed on several occasions. So when you start looking around the code base on create-a-derived-table you may find that code style, project structure or naming conventions don't match the guidance here. It will be a continuos effort from all that use it to slowly conform create-a-derived-table.
+
+## 2-staging
```

Comment:
Not clear what this number relates to - (1) is guide overview but this is nested under a subsequent section